integration-tests/Dockerfile_centos7

```diff
@@ -1,6 +1,8 @@
 FROM centos:7.2.1511
 MAINTAINER Ahmed
 
+SHELL [ "/bin/bash" ]
```

Comment:
I needed to do this to get past https://github.com/docker/for-linux/issues/246 (and I don't understand why, yet; perhaps I have introduced a difference to how the binaries are built that I didn't intend to?), and now I get a failure https://travis-ci.org/github/aelsabbahy/goss/jobs/685298827#L1115 that I'm not sure what do to with :(

Reply:
Yeah, I'll definitely want to understand this better.

Doesn't seem like a correct solution. Maybe try hack the scripts to run a previous version of the goss binary?

Reply:
I agree. I think somewhere I've made a mistake. I'll dig more.

Reply:
Turns out I forgot to specify which package to build. 🙄 🤦